### PR TITLE
Add test for testing openshift-tests binary with master Golang

### PR DESCRIPTION
**Pull request by Rajalakshmi-Girish**
Add tests for testing openshift-tests binary with master Golang

from thread redstack-power/project-mgmt#1608 (comment)
https://github.ibm.com/powercloud/container-dev/issues/1472

Have included `GO_REQUIRED_MIN_VERSION:=` while building as it throws below error with master golang:
```
vendor/github.com/openshift/build-machinery-go/make/targets/golang/../../lib/golang.mk:22: *** `go` is required with minimal version "1.15.2", detected version "go version devel go1.17-c89f1224a5 Mon May 24 22:27:31 2021 +0000 linux/ppc64le". You can override this check by using `make GO_REQUIRED_MIN_VERSION:=`.  Stop.
{"component":"unset","error":"wrapped process failed: exit status 2","file":"/go/test-infra/prow/entrypoint/run.go:80","func":"k8s.io/test-infra/prow/entrypoint.Options.Run","level":"error","msg":"Error executing test process","severity":"error","time":"2021-05-25T04:16:40Z"}
```

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/test-infra/pull/189#" title="Author self-approved">Rajalakshmi-Girish</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **mkumatag** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @mkumatag` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Ftest-infra).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ppc64le-cloud/test-infra/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: config/jobs/ppc64le-cloud/builds/openshift-postsubmit.yaml
Comment (mkumatag) [inline config/jobs/ppc64le-cloud/builds/openshift-postsubmit.yaml:70]: See if we can any other testcases

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: config/jobs/ppc64le-cloud/builds/openshift-postsubmit.yaml
Comment (Rajalakshmi-Girish) [reply config/jobs/ppc64le-cloud/builds/openshift-postsubmit.yaml:70]: We can run the following commands:
```
./openshift-tests images --to-repository=quay.io/powercloud/community-e2e-images
./openshift-tests run kubernetes/conformance --dry-run
./openshift-tests run-upgrade --dry-run --to-image quay.io/openshift/community-e2e-images:e2e-3-k8s-gcr-io-e2e-test-images-sample-apiserver-1-17-4-AC3A0x1ePf4fLoiv all
```
The other commands need a live cluster for execution.
I don't see a --version command. @mkumatag Please let me know if I can include these.

Comment (mkumatag) [reply config/jobs/ppc64le-cloud/builds/openshift-postsubmit.yaml:70]: > We can run the following commands:
> 
> ```
> ./openshift-tests images --to-repository=quay.io/powercloud/community-e2e-images
> ./openshift-tests run kubernetes/conformance --dry-run
> ./openshift-tests run-upgrade --dry-run --to-image quay.io/openshift/community-e2e-images:e2e-3-k8s-gcr-io-e2e-test-images-sample-apiserver-1-17-4-AC3A0x1ePf4fLoiv all
> ```

If these test cases don't live cluster, let's include them or big NO.
> 
> The other commands need a live cluster for execution.
> I don't see a --version command. @mkumatag Please let me know if I can include these.

Comment (Rajalakshmi-Girish) [reply config/jobs/ppc64le-cloud/builds/openshift-postsubmit.yaml:70]: Yes, the above 3 commands don't need cluster as they are only listing images and dry runs.
I shall include and submit the patch after testing.

**Review by mkumatag — APPROVED** [review]
/lgtm

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/test-infra/pull/189#pullrequestreview-669018440" title="LGTM">mkumatag</a>*, *<a href="https://github.com/ppc64le-cloud/test-infra/pull/189#" title="Author self-approved">Rajalakshmi-Girish</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Ftest-infra).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ppc64le-cloud/test-infra/blob/master/OWNERS)~~ [mkumatag]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by ltccci** [comment]
@Rajalakshmi-Girish: Updated the `job-config` configmap in namespace `prow` at cluster `default` using the following files:
 - key `openshift-kubernetes-postsubmit.yaml` using file ``
 - key `openshift-postsubmit.yaml` using file `config/jobs/ppc64le-cloud/builds/openshift-postsubmit.yaml`

<details>

In response to [this](https://github.com/ppc64le-cloud/test-infra/pull/189):

>Add tests for testing openshift-tests binary with master Golang
>
>from thread redstack-power/project-mgmt#1608 (comment)
>https://github.ibm.com/powercloud/container-dev/issues/1472
>
>Have included `GO_REQUIRED_MIN_VERSION:=` while building as it throws below error with master golang:
>```
>vendor/github.com/openshift/build-machinery-go/make/targets/golang/../../lib/golang.mk:22: *** `go` is required with minimal version "1.15.2", detected version "go version devel go1.17-c89f1224a5 Mon May 24 22:27:31 2021 +0000 linux/ppc64le". You can override this check by using `make GO_REQUIRED_MIN_VERSION:=`.  Stop.
>{"component":"unset","error":"wrapped process failed: exit status 2","file":"/go/test-infra/prow/entrypoint/run.go:80","func":"k8s.io/test-infra/prow/entrypoint.Options.Run","level":"error","msg":"Error executing test process","severity":"error","time":"2021-05-25T04:16:40Z"}
>```


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Pull request merged**
